.pre-commit-config.yaml

```diff
@@ -55,14 +55,14 @@ repos:
         types: [markdown]
     -   id: check-merge-conflict
         args: [--assume-in-merge]
+    -   id: check-yaml
+        exclude: \.yamato/*
```

Comment:
were the Yamato configs failing?

Reply:
(discussed offline, templating breaks the checks)

Reply:
(discussed offline, templating breaks the checks)